pkg/converter/convert_unix.go

```diff
@@ -304,9 +312,11 @@ func Pack(ctx context.Context, dest io.Writer, opt PackOption) (io.WriteCloser,
 			}
 		}()
 
-		if _, err := io.Copy(dest, blobFifo); err != nil {
+		buffer := bufPool.Get().(*[]byte)
+		if _, err := io.CopyBuffer(dest, blobFifo, *buffer); err != nil {
```

Comment:
Better to use `defer bufPool.Put(buffer)` after `buffer := bufPool.Get().(*[]byte)`.